spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -24,6 +24,15 @@
           response_body = JSON.parse(response.body, symbolize_names: true)
           expect(response_body[:data].count).to eq 3
         end
+
+        it 'returns metadata with page = 1, per_page = 2 and total_items = 3' do
+          user = Fabricate(:user)
+          Fabricate.times(3, :keyword, user: user)
+          get api_v1_keywords_path, params: { page: 1, per_page: 2 }, headers: create_token_header(user)
+          response_body = JSON.parse(response.body, symbolize_names: true)
+
+          expect(response_body[:meta]).to eq(page: 1, per_page: 2, total_items: 3)
```

Comment:
Should we also assert that the returned `:data` has only 2 items instead of 3? 🤔